w3d_vgui.lua

```diff
@@ -157,7 +168,7 @@ local function CreateButton(sId, x, y, w, h, fcCallback, fcPaint)
     end
 
     -- Register the button in order to make it clickable
-    w3d.tCache.tButtons[sId] = { x = x, y = y, w = w, h = h, fcCallback = fcCallback, iLastTime = SysTime() }
+    w3d.tCache.tButtons[sId] = { x = x, y = y, w = w, h = h, bHovered = bHovered, fcCallback = fcCallback, iLastTime = SysTime(), bNoButton = bNoButton }
```

Comment:
While storing bHovered is not a bad idea, it does not actually change how the code works, so I don't understand why it would fix the issue. I'm using the exact same function in the exact same way above so the result would be the same.